hogvm/typescript/src/stl.ts

```diff
@@ -43,6 +43,34 @@ export const STL: Record<string, (args: any[], name: string, timeout: number) =>
         // eslint-disable-next-line no-console
         console.log(...args)
     },
+    jsonParse: (args) => {
+        return JSON.parse(args[0])
+    },
+    jsonStringify: (args) => {
+        // Recursively convert maps to objects
+        function convert(x: any): any {
```

Comment:
Intrigued to know the perf of this. At some point we probably do want some sort of tests measuring that kind of thing

Reply:
This might have some performance drawbacks, and I hope we get everything else right and can look at benchmarking these things. 

Some thoughts regarding performance for now:
- We might rewrite everything in a compiled language anyway, so a boost here might be 🤷
- `new Map([[key1, value1], [key2, value2]])` and vice versa might be a lot faster?
- Making our own JSONify function (a loop that concatenates strings) may be even faster?

We can also switch HogVM to use objects natively instead of maps. I went with objects initially, but swapped to maps because 1) simple add/remove operations are 10x faster on maps vs objects, 2) objects in JS only support string keys, maps support anything as keys... just like python's dicts. So moving from objects to maps internally made the entire thing faster and more compatible with the Python version. However yes, I didn't think of a potential performance drawback with json.stringify. Let's see how far we can get with this? 🤷